agent.sh

```
@@ -2,15 +2,15 @@
 
 export WORKSPACE_BASE=$(pwd)
 
-sudo docker run \
-    -it \
-    --pull=always \
-    --add-host host.docker.internal:host-gateway \
+docker pull ghcr.io/all-hands-ai/runtime:0.9-nikolaik
+
+docker run -it --pull=always \
+    -e SANDBOX_RUNTIME_CONTAINER_IMAGE=ghcr.io/all-hands-ai/runtime:0.9-nikolaik \
     -e SANDBOX_USER_ID=$(id -u) \
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)